### DISCUSSION TITLE
Look forward to 2022.

### DISCUSSION THREAD

**Discussion by meteos77** [textbrowser/glitch#1] [General]
a program that might interest me after BiblioteQ :-)

**Comment by textbrowser** [comment]
Yup, it'll be a blast.

**Reply by meteos77** [reply]
I already have trouble programming on the command line, so in a graphical version? I'm waiting to see the experience because as you make excellent programs, I expect a real good surprise

**Reply by textbrowser** [reply]
I have hyper-mobility problems. Computers are not made for people. Accessibility is a very interesting and unresolved problem. Typing is not as natural as writing and drawing. Typewriters are awful machines as are most modern keyboards. The science of ergonomics is very sparse and inconclusive. There are many commercial gimmicks. :)

**Reply by meteos77** [reply]
graphic tablets with pen (never tried)
at the library, we have an asus eeepc with touch screen but I didn't have the curiosity to try with biblioteQ

**Reply by textbrowser** [reply]
Hence the Android version for BQ! :)

**Reply by meteos77** [reply]
with the shower for example, it's not a gadget it's really a dedicated tool that works perfectly for its function
I won't talk about biblioteQ here because I'll spoil the forum :-)

**Reply by meteos77** [reply]
my interest for arduino is on this site https://www.locoduino.org/ (sorry it's 100% in french)

**Reply by textbrowser** [reply]
It's a great device. The Pi is also neat. BQ works on a Pi also. :)

**Reply by meteos77** [reply]
I tested biblioteQ on pi400

**Reply by textbrowser** [reply]
Is that the desktop computer?

**Reply by meteos77** [reply]
no just to be up to date :) I think the next pi 500 could be my future desktop computer (it will remind me of Amiga 500)
design ok but missing the numeric keypad :(
I have a very good machine with a : Intel(R) Core(TM) i7-3770K CPU @ 3.50GHz - BiblioteQ is built in < 50s

**Reply by textbrowser** [reply]
The Pi 5 may be available in 2022.

**Reply by textbrowser** [reply]
Having a good development environment is crucial. Compiling on older machines rarely is fine. Developing on them is a nightmare.
